#!/usr/bin/env python
PACKAGE = "aod"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

#       Name               Type      	lvl  	Description               Default    Min   Max
gen.add("calib_corr_R", double_t, 0, "Parametr kalibracyjny skladowej R", 1.0,      0.0, 128.0)
gen.add("calib_corr_G", double_t, 0, "Parametr kalibracyjny skladowej G", 1.0,      0.0, 128.0)
gen.add("calib_corr_B", double_t, 0, "Parametr kalibracyjny skladowej B", 1.0,      0.0, 128.0)
gen.add("auto_calib",	bool_t,   0, "Czy automatycznie kalibrujemy",    False)
gen.add("calib_correction",double_t, 0, "Parametr korekcyjny kalibracji ",0.5,	 0.01, 10.00)

exit(gen.generate(PACKAGE, "aod", "ColorCalib"))
